pom.xml

```diff
@@ -848,6 +853,7 @@
       <modules>
         <module>aot</module>
         <module>aot-maven-plugin</module>
+        <module>aot-precompiled</module>
```

Comment:
I am not 100% sure about the naming (it is _always_ AoT after all); personally I've been using "build-time" or "off-line". But I am approving anyway.

Reply:
renamed to build time everywhere